docs/getting-started.md

```diff
@@ -24,40 +24,47 @@ You can optionally persist the output of the `onit completion` command to your s
 
 ## Usage
 
+### Development Cluster
+
+To setup a cluster for development, after having pushed all the new images to `kind` run:
+```bash
+onit create cluster --set onos-cli.enabled=true
+```
+
 ### Integration Tests
 
 To run a suite of tests, use the `onit run test` command, providing a test image to run:
 
 ```bash
-onit test --image onosproject/onos-tests:latest
+onit test --image <test-image>
+```
+
+For example, to run onos-config suite tests using [kind] cluster:
+```bash
+git clone https://github.com/onosproject/onos-config.git
+cd onos-config
+make kind
+onit test --image onosproject/onos-config-tests:latest --suite gnmi
 ```
 
 Benchmarks can be run with the `onit run benchmark` command:
 
 ```bash
-onit benchmark --image onosproject/onos-tests:latest
+onit benchmark --image <test-image> 
 ```
 
-To run a suite: 
+To run a suite (e.g. a suite of onos-topo tests): 
 ```bash
-onit test --image onosproject/onos-tests:latest --suite config
+onit test --image onosproject/onos-topo-tests:latest --suite topo
 ```
 
-To run a specific test:
+To run a specific test (e.g. an onos-config test):
 ```bash
-onit test --image onosproject/onos-tests:latest --suite config --test TestTransaction
+onit test --image onosproject/onos-config-tests:latest --suite config --test TestTransaction
 ```
 
-### Development Cluster
+More usage examples are provided in [debugging](debugging.md) document.
 
-To setup a cluster for development, after having pushed all the new images to `kind` run:
-```bash
-onit create cluster --set onos-cli.enabled=true
```

Comment:
Andrea said it wasn't possible to get cli working after startup, so it was useful having
`--set onos-cli.enabled=true` in here

Reply:
I didn’t remove it. I just moved it to top of the usages to be the first usage example